sklearn/_loss/_loss.pyx.tp

```diff
@@ -217,12 +217,12 @@ WARNING: Do not edit `sklearn/_loss/_loss.pyx` file directly, as it is generated
 cimport cython
 from cython.parallel import parallel, prange
 import numpy as np
-cimport numpy as np
+cimport numpy as cnp
```

Comment:
Because `cnp` is never used in this file, I think we can remove the import all together:

```suggestion
```

This comment applies to the other files that do not use `cnp`

Reply:
Good point, thanks! I just realized my IDE does not lint Cython files. I'll double check all imports manually.